src/main/java/com/google/codeu/data/User.java

```diff
@@ -0,0 +1,20 @@
+package com.google.codeu.data;
+
+public class User {
```

Comment:
Let's write a javadoc comment for this class. Javadoc comments look like

https://google.github.io/styleguide/javaguide.html#s7-javadoc

/** Some description about the class. */